static/js/components/JobDetails.jsx

```diff
@@ -1,13 +1,33 @@
 var AddForm = React.createClass({
 
   getInitialState: function() {
-    return {clicked: false, errored: false};
+    return {clicked: false, errored: false, JSON_workload_checked: false};
+  },
+
+  raiseJSONWorkloadWarning: function(workload) {
+    if (workload.length === 0 || workload[0] != "{") {
```

Comment:
Should this check if the first char is `"{"` even if there are preceding spaces? (which I think would also cause a parse error?)